start_pokealarm.py

```diff
@@ -85,14 +94,20 @@ def start_server():
     logging.getLogger('connectionpool').setLevel(logging.WARNING)
     logging.getLogger('gipc').setLevel(logging.WARNING)
     logging.getLogger("urllib3").setLevel(logging.WARNING)
+    logging.getLogger('Cache').setLevel(logging.INFO)
```

Comment:
Is the default not INFO?

Reply:
I probably forgot to remove this after testing